zanata-war/src/main/java/org/zanata/action/ValidateEmailAction.java

```diff
@@ -60,6 +61,11 @@
     @In
     EmailChangeService emailChangeService;
 
+    @Create
+    public void onCreate() {
+        identity.checkLoggedIn();
```

Comment:
So, isn't this line doing exactly that? I thought a failed `<restrict>` in `pages.xml` would result in an authorization exception as well.

But, to answer your question, nothing really comes to mind. CDI interceptors can be placed at the class level, in which case they will apply to all methods, but I'm fairly certain this doesn't include the creation of the bean.